packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -369,10 +376,12 @@ export class GarbageCollector implements IGarbageCollector {
         readAndParseBlob: ReadAndParseBlob,
         baseLogger: ITelemetryLogger,
         existing: boolean,
-        metadata?: IContainerRuntimeMetadata,
+        metadata: IContainerRuntimeMetadata | undefined,
+        private readonly summarizerClient: boolean = true,
```

Comment:
This is interesting, I'm assuming you're doing this for testing

Reply:
It's for filtering out the inactive object logs.

Reply:
I meant defaulting `summarizerClient` to true. Also, maybe we want to name this `isSummarizerClient`?

Reply:
Ah yes, that's a slight optimization for testing :-).

Reply:
Renamed to `isSummarizerClient`